README.md

```diff
@@ -12,11 +12,11 @@ When compiling `LND`, make sure that the peersrpc and signerrpc services are ena
 
 In order to successfully connect to `LND`, we need to pass in the grpc address and authentication credentials. These values can be passed in via the command line when running the `LNDK` program, like this:
 
-`cargo run -- <ADDRESS> <TLSPATH> <MACAROONPATH>`
+`cargo run -- --address=<ADDRESS> --cert=<TLSPATH> --macaroon=<MACAROONPATH>`
 
 Or in a more concrete example:
 
-`cargo run -- https://localhost:10009 /home/<USERNAME>/.lnd/tls.cert /home/<USERNAME>/.lnd/data/chain/bitcoin/regtest/admin.macaroon`
+`cargo run -- --address=https://localhost:10009 --cert=/home/<USERNAME>/.lnd/tls.cert --macaroon=/home/<USERNAME>/.lnd/data/chain/bitcoin/regtest/admin.macaroon`
```

Comment:
Add a section here explaining that we can also specify a config file?

Reply:
Also `--help` works.

Reply:
Thanks